src/WebJobs.Script/Description/Rpc/WorkerLanguageInvoker.cs

```diff
@@ -61,6 +61,7 @@ internal class WorkerLanguageInvoker : FunctionInvokerBase
                 ExecutionContext = context.ExecutionContext,
                 Inputs = inputs,
                 ResultSource = new TaskCompletionSource<ScriptInvocationResult>(),
+                AsyncExecutionContext = System.Threading.ExecutionContext.Capture(),
```

Comment:
This was previously being set further down in the pipeline. My multi-invocation node tests were failing as the first execution did not always have the correct context. With this change, they all pass.